testdata/generated/search-category-infrastructure-subcategories.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "name": "longdocs",
+    "title": "Long Docs",
+    "version": "1.0.4",
+    "release": "ga",
+    "description": "This integration contains pretty long documentation.\nIt is used to show the different visualisations inside a documentation to test how we handle it.\nThe integration does not contain any assets except the documentation page.\n",
+    "type": "integration",
+    "download": "/epr/longdocs/longdocs-1.0.4.zip",
+    "path": "/package/longdocs/1.0.4",
+    "icons": [
+      {
+        "src": "/img/icon.svg",
+        "path": "/package/longdocs/1.0.4/img/icon.svg",
+        "type": "image/svg+xml"
+      }
+    ],
+    "conditions": {
+      "kibana": {
+        "version": ">6.7.0"
+      },
+      "elastic": {
+        "subscription": "gold"
+      }
+    },
+    "categories": [
+      "custom",
+      "web"
```

Comment:
These packages don't have the `infrastructure` category, but appear in results because `web` is a subcategory of `infrastructure`.